schema/meter_provider.json

```diff
@@ -46,6 +46,12 @@
             "properties": {
                 "exporter": {
                     "$ref": "#/$defs/MetricExporter"
+                },
+                "producers": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/$defs/MetricProducer"
+                    }
                 }
```

Comment:
Why is this needed here if already in the top level MetricReader? If it is required for pull (maybe to control at a granular level) why not for periodic as well?

Reply:
https://github.com/open-telemetry/opentelemetry-configuration/pull/93

Reply:
Thanks for calling this out @bogdandrutu!